plugins/armorblox/help.md

```diff
@@ -0,0 +1,163 @@
+# Description
+
+Armorblox secures enterprise communications over email and other cloud office applications with the power of Natural Language Understanding. The Armorblox platform connects over APIs and analyzes thousands of signals to understand the context of communications and protect people and data from compromise. Over 56,000 organizations use Armorblox to stop BEC and targeted phishing attacks, protect sensitive PII and PCI, and automate remediation of user-reported email threats. Armorblox was featured in the 2019 Forbes AI 50 list and was named a 2020 Gartner Cool Vendor in Cloud Office Security. Founded in 2017, Armorblox is headquartered in Sunnyvale, CA and backed by General Catalyst and Next47
+
+# Key Features
+
+* Fetches incidents detected by Armorblox for the given tenant.
+* Retrieves the remediation action for a given incident.
+
+# Requirements
+
+* Requires an API key from the product.
+
+# Supported Product Versions
+
+* 1.0.0
+
+# Documentation
+
+## Setup
+
+The connection configuration accepts the following parameters:
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|api_key|credential_secret_key|None|True|Armorblox API Key|None|9de5069c5afe602b2ea0a04b66beb2c0|
+|tenant_name|string|None|True|Armorblox Tenant Name|None|my-tenant-name|
+
+Example input:
+
+```
+{
+  "api_key": "9de5069c5afe602b2ea0a04b66beb2c0",
+  "tenant_name": "my-tenant-name"
+}
+```
+
+## Technical Details
+
+### Actions
+
+#### Get Remediation Action
+
+This action is used to get remediation action of a incident identified by Armorblox.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|incident_id|string|None|True|An integer number representing the incident|None|3490|
+
+Example input:
+
+```
+{
+  "incident_id": 3490
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|remediation_details|string|True|Remediation Action of the requested incident identified by Armorblox|
+
+Example output:
+
+```
+```
+
+### Triggers
+
+#### Get Incidents
+
+This trigger is used to get a list of incidents identified by Armorblox, by default it starts polling all the incidents since last day.
```

Comment:
```suggestion
This trigger is used to get a list of incidents identified by Armorblox. By default, it starts polling all the incidents since last day.
```